daemon/daemon.go

```diff
@@ -445,8 +445,8 @@ func (daemon *Daemon) setupResolvconfWatcher() error {
 			select {
 			case event := <-watcher.Events:
 				if event.Name == "/etc/resolv.conf" &&
-					(event.Op&fsnotify.Write == fsnotify.Write ||
-						event.Op&fsnotify.Create == fsnotify.Create) {
+					(event.Op&fsnotify.Write != 0 ||
+						event.Op&fsnotify.Create != 0) {
```

Comment:
@tiborvass I am getting the following error when I try to do ```make binary```

```daemon/daemon.go:448: invalid operation: event.Name == "/etc/resolv.conf" && event.Op & fsnotify.Write (mismatched types bool and fsnotify.Op)```

I have the following edit that you requested...

```go
if event.Name == "/etc/resolv.conf" &&
	(event.Op & (fsnotify.Write | fsnotify.Create)) {
```